src/client/lib/PacActivityBarUI.ts

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as vscode from 'vscode';
+import { AdminEnvironmentListing, AuthProfileListing, SolutionListing, PacOutputWithResultList } from '../pac/PacTypes';
+import { PacWrapper } from '../pac/PacWrapper';
+
+export function RegisterPanels(pacWrapper: PacWrapper): vscode.Disposable[] {
```

Comment:
in a next PR, do refactor a bit, e.g. have one module for the panel registration func, and another class file for the tree views